src/components/Sections/SectionTable.less

```diff
@@ -10,6 +11,7 @@
   }
 
   th, td {
+    padding: 3px !important;
```

Comment:
why do we need this ?

Reply:
override the default padding that the ui library adds there (it was larger and we need more space to show the content better)